index.rst

```diff
@@ -1,52 +1,163 @@
+- :math:`N_\mathrm{star}`: the number of stars used to fit the PSF (note: in general, :math:`N_\mathrm{star} N_\mathrm{PSF} \ll N_\mathrm{pixels}`)
+- :math:`N_\mathrm{cells}`: the number of cells used to model PSF spatial variations
 
-.. Add content below. Do not include the document title.
+.. _psffit-general:
 
-.. note::
+General-Purpose Software
+========================
 
-   **This technote is not yet published.**
+.. _psffit-general-psfex:
+
+PSFEx
+-----
+
+`PSFex <http://www.astromatic.net/software/psfex>`_ is a generic PSF fitting tool last updated in 2014. While intended to be run as a standalone command-line program, the LSST Stack's ``meas_extensions_psfex`` provides a wrapper that lets its core functionality be used as a ``PsfDeterminer``.
+
+.. _PSFex Manual: http://psfex.readthedocs.io/en/latest/
+
+When run as a command-line tool, PSFex identifies PSF template sources by filtering an input catalog. The primary criterion is to find a locus of sources with constant half-light-radius as a function of magnitude, but the user may specify cuts on SNR, arbitrary flags, image ellipticity, or presence of zero-weight pixels. Crowded fields are handled by iteratively computing a PSF model, modeling each identified source, and removing 4-σ outliers (`PSFex Manual`_). It is not clear how high a source density is supported by this approach, but clearly there must be *some* isolated stars for the PSFEx approach to be viable.
+
+The Stack's version does not use the above system, instead providing an independent ``StarSelector`` that uses similar but manually specified filtering criteria, and no crowded-field handling.
+
+PSFs are fit as a sum of basis functions, which may be pixel/nonparametric (as used by ``meas_extensions_psfex``), Gauss-Laguerre, or a custom basis. The output PSF is always presented as a supersampled image; the developers concede that this approach does not handle PSF wings or diffraction spikes well.
+
+The pixel-basis fitter constructs an image using "sinc interpolation" (essentially kernel smoothing by a Lanczos4 function) to resample the model to the pixel grid around each star. The fit is computed by minimizing the χ\ :sup:`2` of the interpolated model across all sources.
+
+PSFex fits PSF variations simultaneously with the PSF itself. Variability may be taken with respect to source position, source parameters, or image metadata (while PSFex itself requires that this input be in SExtractor and FITS format, respectively, ``meas_extensions_psfex`` does the appropriate translation from Sources and Exposures). PSF variations are represented by a low-order polynomial whose "coefficients" are themselves PSF images. There is no support for high-frequency components of the type discussed by [CMJ2012]_. The algorithm requires :math:`\mathcal{O}((N_\mathrm{PSF} N_\mathrm{coeff})^3 + N_\mathrm{PSF}^2 N_\mathrm{star} N_\mathrm{coeff})` time, where :math:`N_\mathrm{coeff}` is the number of terms in the polynomial.
```

Comment:
For your first question, I think they mean fitting multiple images simultaneously. They provide some examples of relevant image metadata:
> Although PSF variations are more likely to depend essentially on source position on the focal plane, it is thus possible to include explicit dependency on parameters such as telescope position, time, source flux (Fig. 5.7) or instrument temperature.

I'm not sure I understand the second question. Why would binning be necessary for non-position parameters? It's still a multivariate polynomial, just with more terms.